target/linux/ath79/base-files/etc/board.d/02_network

```diff
@@ -49,6 +49,12 @@ ath79_setup_interfaces()
 			"0@eth0" "1:lan" "2:lan" "3:lan" "4:lan" "5:wan"
 		;;
 
+	"pcs,cr3000")
+		ucidef_set_interfaces_lan_wan "eth0.1" "eth1"
```

Comment:
Would ucidef_set_interfaces_wan be sufficient here, since switch is defined below?

Reply:
> Would ucidef_set_interfaces_wan be sufficient here, since switch is defined below?

Correct. And keeping the real interface order in mind it should be:

```
ucidef_add_switch "switch0" \
	"0@eth1" "1:lan:1" "2:lan:4" "3:lan:3" "4:lan:2"
ucidef_set_interface_wan eth0
;;
```

Reply:
ah, I did not know that.